composer.json

```diff
@@ -59,8 +59,25 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"test": [
+			"@lint",
+			"@check-cs",
+			"@run-tests",
+			"@check-complete-strict"
 		]
 	},
+	"scripts-descriptions": {
+		"lint": "Lint PHP files against parse errors.",
+		"lint-ci": "Lint PHP files against parse errors, but output the results as Checkstyle XML (used for CI purposes).",
+		"check-cs": "Run the PHPCS script against the entire codebase.",
+		"fix-cs": "Run the PHPCBF script to fix all the autofixable violations on the codebase.",
+		"install-codestandards": "Manually install the coding standards, so that they are discoverable by PHPCS script. Used mostly for CI purposes",
```

Comment:
Do we still need the `install-codestandards` script ? If not, now would be the time to remove it.

Reply:
I've found the plugin to work really well, and haven't had to run this script manually for longer than I can remember. At the very least, a `composer install` or `composer update` is enough to trigger it anyway.

Let's scrap it, to avoid any confusion.